angular-ngrx-scss/src/app/state/user/user.state.ts

```diff
@@ -50,3 +51,8 @@ export interface UserApiResponse {
   created_at: string;
   updated_at: string;
 }
+
+export enum ProfileType {
+  ORGANIZATION = 'Organization',
+  USER = 'User',
+}
```

Comment:
style: PascalCase is generally used for enum member names. All caps is generally reserved for global constants instead. TypeScript doesn't have an official style guide to reference here, but all of their examples and most production code I've seen uses PascalCase in this context.